source/_integrations/yandextts.markdown

```diff
@@ -9,6 +9,12 @@ ha_domain: yandextts
 
 The `yandextts` text-to-speech platform uses [Yandex SpeechKit](https://tech.yandex.com/speechkit/) Text-to-Speech engine to read a text with natural sounding voices.
 
+<div class='note warning'>
+This integration is working only with old API keys.
+  
+For the new API keys this intergration is not working. There is a custom component that is working with new API keys [Yandex-tts-speechkit-FIX](https://github.com/tayanov/Yandex-tts-speechkit-FIX).
+</div>
```

Comment:
This PR breaks the layout as it mixes Markdown & HTML without switching context.

A new line has to be added after the opening div and before the closing div.